app/src/main/java/org/dhis2/usescases/eventsWithoutRegistration/eventDetails/models/EventInputDateUiModel.kt

```diff
@@ -14,4 +14,5 @@ data class EventInputDateUiModel(
     val showField: Boolean = true,
     val is24HourFormat: Boolean = true,
     val selectableDates: SelectableDates? = null,
+    val onError: (() -> Unit)? = null,
```

Comment:
It is not a good idea to include lambdas in data classes. We should discuss it with the group and decide.